R-package/R/lgb.Booster.R

```diff
@@ -1346,6 +1354,8 @@ lgb.save <- function(booster, filename, num_iteration = NULL) {
 #' @examples
 #' \donttest{
 #' library(lightgbm)
+#' \dontshow{setLGBMthreads(2L)}
+#' \dontshow{data.table::setDTthreads(1L)}
```

Comment:
Per https://cran.r-project.org/doc/manuals/R-exts.html

> ... `\dontshow{}` for extra commands for testing that should not be shown to users, but will be run by `example()`

These `\dontshow{}` blocks hide this code from users, but ensure it runs when CRAN checks the package.

Thanks to @jangorecki for the suggestion (https://github.com/Rdatatable/data.table/issues/5658#issuecomment-1741934995).